src/main/kotlin/lotto/domain/Cash.kt

```diff
@@ -0,0 +1,9 @@
+package lotto.domain
+
+data class Cash(val amount: Int) {
+    init {
+        require(amount >= 0)
+    }
+
+    fun pay(amount: Int): Cash = Cash(this.amount.minus(amount))
```

Comment:
```suggestion
    fun pay(amount: Int): Cash = this.copy(amount = this.amount - amount)
```

data class는 copy 메소드가 있어요 : ) 

요번 기회에 copy method에 대해서 공부해볼까요~?

[[Kotlin] 깊은 복사(Deep Copy) 하는 3가지 방법](https://velog.io/@dddooo9/Kotlin-%EA%B9%8A%EC%9D%80-%EB%B3%B5%EC%82%ACDeep-Copy-%ED%95%98%EB%8A%94-3%EA%B0%80%EC%A7%80-%EB%B0%A9%EB%B2%95)